### PR TITLE
LP-2710 Enable edX pre reqs courses and integrate pre reqs on course about page

### PR DESCRIPTION
**Pull request by aliassad**
Enable edX pre reqs courses and integrate pre reqs on course about page
[adg-edx-theme PR](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/300)

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1958/workflows/workflows_id/jobs/1958/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/courseware_override/helpers.py (100%)
openedx/adg/lms/courseware_override/tests/test_helpers.py (100%)
-------------
Total:   9 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/courseware_override/tests/test_helpers.py [hunk 1]
```diff
@@ -72,12 +72,15 @@ def test_get_language_name_from_codes(language_codes, language_names):
 @pytest.mark.django_db
 @mock.patch('openedx.adg.lms.courseware_override.helpers.is_testing_environment')
 @mock.patch('openedx.adg.lms.courseware_override.helpers.get_course_instructors')
+@mock.patch('openedx.adg.lms.courseware_override.helpers.get_pre_requisite_courses_not_completed')
 @mock.patch('openedx.adg.lms.courseware_override.helpers.get_language_names_from_codes')
-def test_get_extra_course_about_context(mock_get_lang_names, mock_get_instructors, mock_testing_env):
+def test_get_extra_course_about_context(mock_get_lang_names, mock_get_pre_requisite_courses_not_completed,
+                                        mock_get_instructors, mock_testing_env, request):
```
Comment (asadali145) [inline openedx/adg/lms/courseware_override/tests/test_helpers.py:78]: Lets please break lines as per edX guidelines.

Comment (aliassad) [reply openedx/adg/lms/courseware_override/tests/test_helpers.py]: Done

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1961/workflows/workflows_id/jobs/1961/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/courseware_override/helpers.py (100%)
openedx/adg/lms/courseware_override/tests/test_helpers.py (100%)
-------------
Total:   9 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1962/workflows/workflows_id/jobs/1962/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/courseware_override/helpers.py (100%)
openedx/adg/lms/courseware_override/tests/test_helpers.py (100%)
-------------
Total:   9 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by javeriaejaz-arbisoft — APPROVED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1963/workflows/workflows_id/jobs/1963/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/courseware_override/helpers.py (100%)
openedx/adg/lms/courseware_override/tests/test_helpers.py (100%)
-------------
Total:   9 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by asadali145 — APPROVED** [review]

**Pull request merged**